### PR TITLE
Explained difference between scope & class method

### PR DESCRIPTION
**Pull request by ghost**
### Summary

Today in the Ruby on Rails slack, the question arose what the difference between a scope and a class method is.

After googling I found an accepted stack overflow answer which I partly copied in order to answer the question.

This part copy of mine included the wrong quote from this guide, that a scope and a class method is **exactly** the same. This is simply not true.

### CODE REVIEW

**Comment by LemonAndroid** [comment]
Example for false information being spread based on this guide: https://stackoverflow.com/questions/32930312/ruby-on-rails-activerecord-scopes-vs-class-methods

**Comment by schneems** [comment]
Thanks!

File: guides/source/active_record_querying.md [hunk 1]
```diff
@@ -1277,14 +1277,20 @@ class Article < ApplicationRecord
 end
 ```
 
-This is exactly the same as defining a class method, and which you use is a matter of personal preference:
+This is almost the same as defining a class method, except for that scopes always return an `ActiveRecord::Relation` object.
```
Comment (kamipo) [inline guides/source/active_record_querying.md]: **exactly** the same is not true, but the remaining part is almost the same with here:

https://github.com/rails/rails/blob/97feb4996b1c88f770101dfce6d4d3a6baf6bb33/guides/source/active_record_querying.md#L1360-L1370

Comment (bogdanvlviv) [reply guides/source/active_record_querying.md]: How about removing this part in favor of the info below in order to not repeat yourself?

Comment (kamipo) [reply guides/source/active_record_querying.md]: Yeah, it is a good idea to me.

**Pull request merged**